google/cloud/bigquery/dataset.py

```diff
@@ -764,6 +765,25 @@ def default_encryption_configuration(self, value):
             api_repr = value.to_api_repr()
         self._properties["defaultEncryptionConfiguration"] = api_repr
 
+    @property
+    def is_case_insensitive(self):
+        """Optional[bool]: TRUE if the dataset and its table names are case-insensitive, otherwise FALSE.
```

Comment:
nit: use `True` and `False`, following Python convention

Reply:
Changed to `True` and `False`